<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>350</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Sign up </string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveWarning">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>650</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>This image will be used for authentication later, Keep it safe!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
box-shadow: none;
border-style:flat;
text-align: center;
font: 12pt &quot;MS Shell Dlg 2&quot;;
border: 0.5px solid #000000;</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordToQR">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 179, 30);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="progessEmail">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 50px;
width: 50px;
background-color: rgb(0, 179, 30);
border-radius: 55%;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="progressQR">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  height: 50px;
  width: 50px;
  background-color: rgb(0, 170, 255);
  border-radius: 55%;</string>
   </property>
   <property name="text">
    <string>QR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="progessPassword">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">  height: 50px;
  width: 50px;
background-color: rgb(0, 179, 30);
  border-radius: 55%;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmailToPassword">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 179, 30);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>EmailToPassword</zorder>
  <zorder>saveButton</zorder>
  <zorder>browseButton</zorder>
  <zorder>imageLabel</zorder>
  <zorder>label</zorder>
  <zorder>saveWarning</zorder>
  <zorder>homeButton</zorder>
  <zorder>PasswordToQR</zorder>
  <zorder>progessEmail</zorder>
  <zorder>progressQR</zorder>
  <zorder>progessPassword</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
